### PR TITLE
Issue #4627 Fixed: "Fixing the Resizing of the 'Add a Source' Drag Down on the Dashboard"

### PR DESCRIPTION
**Pull request by pronav5000**
The 'Add a Source' menu now properly resizes with a scrollbar when the box is shrunk. Here is an image depicting the change:
image

ORIGINAL BUG REPORT: https://github.com/skyportal/skyportal/issues/4627

### CODE REVIEW

**Comment by Theodlz** [comment]
please @pronav5000 update your PR's branch so the github actions can run (I just fixed them)

**Comment by pronav5000** [comment]
> please @pronav5000 update your PR's branch so the github actions can run (I just fixed them)

submitted another push, does that fix the problem?

**Comment by mcoughlin** [comment]
@pronav5000 https://github.com/pronav5000/skyportal/tree/main there should be a button that says "sync fork", please do that.

**Comment by pronav5000** [comment]
> @pronav5000 https://github.com/pronav5000/skyportal/tree/main there should be a button that says "sync fork", please do that.

I believe I did that, most of the checks are passing now. Anything I need to do to pass the last one?

**Comment by Theodlz** [comment]
No! Looks good. LGTM

File: nohup.out [hunk 1]
```diff
@@ -0,0 +1,84 @@
+sudo: apt-mirror: command not found
```
Comment (Theodlz) [inline nohup.out:1]: Oh actually, please remove this file. I suggest not running `git add .` so that doesnt happen next time.

**Review by Theodlz — CHANGES_REQUESTED** [review]
Please remove that extra file and we're done!

**Comment by Theodlz** [comment]
@pronav5000 also remove the changes you made to package.json. they are unnecessary

**Comment by Theodlz** [comment]
@pronav5000

**Comment by Theodlz** [comment]
Closing as we removed the widget from the dashboard.

**Pull request closed without merging**